contrib/ansible/roles/skydive_common/tasks/main.yml

```diff
@@ -1,6 +1,7 @@
 - name: Display all variables/facts known for a host
   debug:
     var: hostvars[inventory_hostname]
+  when: display_all_variables|default(false)|bool == true
```

Comment:
I would prefer to use `verbosity`. it seems to handle what you try to introduce nicely. What do you think ?
https://docs.ansible.com/ansible/latest/modules/debug_module.html

Reply:
```- name: Display all variables/facts known for a host
  debug:
    var: hostvars[inventory_hostname]
    verbosity: 4```

Is it ok?

Reply:
yes